app/views/application_letters/show.html.erb

```diff
@@ -3,13 +3,19 @@
   <h1><%=t '.title', :default => model_class.model_name.human.titleize %></h1>
 </div>
 
+<h3><%= @application_letter.user.profile.name %></h3>
+
 <dl class="dl-horizontal">
-  <dt><strong><%= model_class.human_attribute_name(:motivation) %>:</strong></dt>
-  <dd><%= @application_letter.motivation %></dd>
-  <dt><strong><%= model_class.human_attribute_name(:user_id) %>:</strong></dt>
-  <dd><%= @application_letter.user_id %></dd>
-  <dt><strong><%= model_class.human_attribute_name(:event_id) %>:</strong></dt>
-  <dd><%= @application_letter.event_id %></dd>
+  <dt><strong><%= Profile.human_attribute_name(:gender) %>:</strong></dt>
```

Comment:
Should all dt-elements be strong? Maybe the style could than be adjusted in a CSS-file?